src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java

```diff
@@ -226,7 +226,7 @@ private byte[] getCredentialsBypassCache() {
                     Instant expiry = Instant.parse(expiryString);
 
                     if (expiry.isBefore(Instant.now(clock))) {
-                        String responseString = "TES responded with expired credentials: " + credentials;
```

Comment:
Let's keep this somewhat useful. Add in the expiration date to the message at least.

Reply:
added